### PR TITLE
Removes v0 location transformers 

### PR DESCRIPTION
**Pull request by ryanshaw**
## Summary

This PR closes a number of open issues removing unused v0 location transformer functions. It was easier and more efficient to remove this all at once.

The functions are:

- `isFacilityOpenAllDay`
- `isFacilityClosed`
- `transformOperatingHours`
- `hasCovidPhoneNumber`
- `parseHours`
- `transformFacilities`
- `transformCommunityProvider`
- `transformSettings`
- `transformParentFacilities`
- `transformFacility`

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53138
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53142
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53144
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53145
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53137
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/53139

## Testing done

- Unit testing
- e2e testing

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/vaos/services/appointment/transformers.js [hunk 1]
```diff
@@ -181,9 +180,9 @@ function setVideoData(appt) {
     kind: videoData.appointmentKind,
     url: getVideoVisitLink(appt),
     isAtlas: !!videoData.tasInfo,
-    atlasLocation: videoData.tasInfo
-      ? transformATLASLocation(videoData.tasInfo)
-      : null,
+    // atlasLocation: videoData.tasInfo
```
Comment (ryanshaw) [inline src/applications/vaos/services/appointment/transformers.js:183]: This is just to allow unit tests to pass. All v0 appointment transformer functions were removed in another PR that has not been merged yet: https://github.com/department-of-veterans-affairs/vets-website/pull/25018

File: src/applications/vaos/services/appointment/transformers.v2.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import {
   PURPOSE_TEXT_V2,
 } from '../../utils/constants';
 import { getTimezoneByFacilityId } from '../../utils/timezone';
-import { transformFacilityV2 } from '../location/transformers.v2';
+import { transformFacilityV2 } from '../location/transformers';
```
Comment (ryanshaw) [inline src/applications/vaos/services/appointment/transformers.v2.js:10]: This is just to allow unit tests to pass. All v0 appointment transformer functions were removed in another PR that has not been merged yet: https://github.com/department-of-veterans-affairs/vets-website/pull/25018

**Comment by ryanshaw** [comment]
This PR will need to be carefully merged after https://github.com/department-of-veterans-affairs/vets-website/pull/25018 is merged in. Might need to make a few changes after the other PR is merged in as well.

File: src/applications/vaos/services/appointment/transformers.js (hunk 1 above)
Comment (ryanshaw) [reply src/applications/vaos/services/appointment/transformers.js]: Removed this after merging in changes from `main`

File: src/applications/vaos/services/appointment/transformers.v2.js (hunk 2 above)
Comment (ryanshaw) [reply src/applications/vaos/services/appointment/transformers.v2.js]: Removed this after merging in changes from `main`

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**